<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_window</class>
 <widget class="QWidget" name="new_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonregister">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdituser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入您的用户名（必填）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditpassword">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入您的密码（必填）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditpasswordconfirm">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请再次输入您的密码（必填）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditemail">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入您的电子邮箱（选填）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditphone">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="placeholderText">
    <string>请输入您的电话号码（必填）</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeluser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名(必填）</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelpassword">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelpasswordconfirm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelemail">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelsex">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxsex">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>秘密</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelphone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>电话号码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
